public/app/features/inspector/QueryInspector.tsx

```diff
@@ -237,7 +237,7 @@ export class QueryInspector extends PureComponent<Props, State> {
           </p>
         </div>
         {this.renderExecutedQueries(executedQueries)}
-        <div className={styles.toolbar}>
+        <Stack direction={'row'} gap={2} justifyContent={'flex-start'} wrap>
```

Comment:
```suggestion
        <Stack direction={'row'} gap={1} justifyContent={'flex-start'} wrap>
```

Maybe this is a personal preference, but I think it looks better with 1. Any thoughts @EdPoole ?

Reply:
I wanted to keep the space we have currently, but @EdPoole, if you also think so I can change it 🤷‍♀️

Reply:
You can merge without changing this :)